### DISCUSSION TITLE
PlanetScale + Prisma for read-only regions

### DISCUSSION THREAD

**Discussion by suptejas** [planetscale/discussion#406] [Q&A] [answered]
We've setup multiple read-only regions with PlanetScale - we currently use Prisma as our ORM. Are there any guides or recommendations on how to use read-only regions with Prisma based on the user's location?

If this is not possible using Prisma, are there any workarounds you can recommend?

**Comment by janpio** [comment, marked as answer]
Prisma can currently not natively handle read-only replicas unfortunately: https://github.com/prisma/prisma/issues/172

You will have to instantiate a second Prisma Client that is explicitly configured to talk to the replica, and then either manually or in some automated way decide when to use which Client. The issue above has some examples from users, or we have an example middleware as a starting point here: https://github.com/prisma/prisma-read-replica-middleware The new Prisma Client extensions preview feature might offer a better way to implement this even.

**Reply by fooforge** [reply]
Thanks for chiming in, @janpio! :-)
